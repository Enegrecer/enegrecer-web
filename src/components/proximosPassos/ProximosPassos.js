import React from 'react';

const ProximosPassos = () => (
  <div>
    <h1>Cadastro Realizado com Sucesso  </h1>
    <p>Texto de Pr√≥ximos passos! </p>

  </div>
);

export default ProximosPassos;
