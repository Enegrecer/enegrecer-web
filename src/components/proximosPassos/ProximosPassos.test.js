import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import ProximosPassos from './ProximosPassos'

describe('ProximosPassos', () => {
  const wrapper = shallow(<ProximosPassos />);

  it('deve possuir a mesma estrutura', () => {
    const tree = toJson(wrapper);
    expect(tree).toMatchSnapshot();
  });
});

