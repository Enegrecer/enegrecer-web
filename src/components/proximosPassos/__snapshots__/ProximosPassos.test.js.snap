// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProximosPassos deve possuir a mesma estrutura 1`] = `
<div>
  <h1>
    Cadastro Realizado com Sucesso  
  </h1>
  <p>
    Texto de Pr√≥ximos passos! 
  </p>
</div>
`;
